bitcoinology/what-is-lightning-network.md

```diff
@@ -0,0 +1,90 @@
+    It was originally tended to be very apolitical. And So you can think of Bitcoin like that or like hard currency in the old Soviet Union. What you can't really think of it as is like a replacement for Visa because it just doesn't scale to that. It doesn't have the right privacy properties. It doesn't have the right consumer experience. So that's why Me and quite a few people were quite negative about this viewpoint. On the other hand, there were people like this, the most obvious example being Roger Ver and several others, Charlie Shrem was another one, who were very public figures in 2013, very popular and famous. And a lot of the time they were going on the media and they were trying to push this narrative. Bitcoin is gonna be like the next Visa.
+    
+     We started talking about transactions per second metrics. And they were always trying to play down the idea that there would be fees and there would be delays. They all know we can accept unconfirmed transactions and we can just kind of work around it. And even whole companies were built up around this concept that we could make these transactions kind of sort of free and kind of sort of instant, but actually they weren't really. And this whole, well, I suppose the guy at the bottom here kind of perfectly illustrates the end of that period, or the kind of transitionary period. Because that was an amazing video, by the way, in 2013 with him sitting on this bouncy ball. I have no idea to this day why he was sitting on that when they were talking to him. But this is Mark Karpellis, he used to run Mt. Gox or MT Gox. And of course, he's another sort of aspect of the kind of the problems of those early era because there was a lot of, you could call it, amateurishness or unprofessionalness in maintaining these very serious exchange websites. And famously, it completely collapsed with millions of dollars lost, or millions of...hundreds of millions, I suppose. Charlie Shrem was a businessman. This sign here is typical of that era. That was an early attempt to, like, look, we can actually, you know, exchange Bitcoin in shops and bars and things. It's a bit perhaps strange for me to put Andreas Antonopoulos on the same slide, but he's just like another aspect of that same early days where they were talking about, a lot of enthusiasm about Bitcoin as a consumer payment system. He's more on the developer and engineer side, but to this day he's still an important figure 
+
+## The next few years
```

Comment:
This heading breaks up the paragraphs. Can we find a better break to put the header?